go.mod

```diff
@@ -3,7 +3,7 @@ module cloud.redhat.com/clowder/v2
 go 1.13
 
 require (
-	github.com/RedHatInsights/strimzi-client-go v0.21.1-2
+	github.com/RedHatInsights/strimzi-client-go v0.21.1-3
```

Comment:
Do we need this minor update or did this sneak. Go has some sneaky libraries :upside_down_face:

Reply:
Needed for KafkaConnect

I am keeping the version name of his library (0.21.1) to match the version of Strimzi operator. I am just bumping the `-N` if I make changes to the package if its still based on Strimzi v0.21.1